models/articles.models.js

```diff
@@ -1,16 +1,24 @@
-const db = require('../db/connection.js')
+const db = require("../db/connection.js");
 
-function selectArticleById(article_id){
-    return db
-    .query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
+function selectAllArticles() {
+  return db
+    .query(
+      "SELECT articles.*, count(comments.article_id) as comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY created_at DESC"
```

Comment:
To make comment_count a number instead of string, just need to add `:: INT` before `AS comment_count`